operations/helm/tests/enterprise-https-values-generated/mimir-distributed/templates/compactor/compactor-statefulset.yaml

```diff
@@ -133,3 +133,5 @@ spec:
               drop:
               - ALL
             readOnlyRootFilesystem: true
+          env:
+            []
```

Comment:
Note for the reviewers, the side effect of the changes is that the components render a default empty env, when there is no environment variables. This is valid; I don't have issues with that.